### PR TITLE
Added Swagger docs healthiness spec #1607

### PR DESCRIPTION
**Pull request by dinesh-skyach**
Added Swagger docs healthiness spec #1607

### CODE REVIEW

File: spec/api/swagger_spec.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# encoding: UTF-8
+# frozen_string_literal: true
+
+describe 'Swagger', type: :request do
+
+  it "return APIv2 swagger docs" do
+    get "/api/v2/swagger"
```
Comment (ysv) [inline spec/api/swagger_spec.rb]: What do you think about adding `expect{}.not_to raise_error`

Comment (dinesh-skyach) [reply spec/api/swagger_spec.rb]: @ysv done.

Comment (ysv) [reply spec/api/swagger_spec.rb]: @dinesh-skyach I think that we need to check that in doesn't raise_error and status is 200

Comment (dinesh-skyach) [reply spec/api/swagger_spec.rb]: @ysv ok will keep both.

**Review by ysv — APPROVED** [review]

**Comment by mod** [comment]
@dinesh-skyach tests fails

**Comment by ysv** [comment]
@dinesh-skyach rebase please

**Pull request merged**